src/alerts/check_signals_updates.R

```diff
@@ -0,0 +1,207 @@
+box::use(jsonlite[fromJSON, toJSON])
+box::use(httr[GET, POST, add_headers, status_code, content])
+box::use(purrr)
+box::use(logger[log_error, log_info])
+
+box::use(../utils/get_env[get_env])
+box::use(../utils/hs_logger)
+box::use(cs = ../utils/cloud_storage)
+
+hs_logger$configure_logger()
+
+
+#' Queries the GitHub API to get all runs from a given monitoring workflow
+#' Requires workflows to be named following this convention: `monitor_<indicator_id>.yaml`
+#'
+#' NOTE: More data will be returned as each workflow is run more times. Perhaps won't scale well,
+#' depending on how many runs a workflow will have
+#'
+#' @param indicator_id ID of the indicator
+#'
+#' @returns JSON API response
+gh_status <- function(indicator_id) {
+  # TODO Temp cleaning before workflows are renamed to match indicator ids
+  ind <- sub("^[^_]*_", "", indicator_id)
```

Comment:
`sub()` and the base family of regex functions are perfectly fine to work with. Just note we have `{stringr}` in our current imports, which aims to be a much cleaner family of functions for this purpose. Basically, it has clearer function naming, like `str_detect()`, `str_extract()`, `str_remove()`, and a consistent API of the vector argument passed first as `x`, `pattern` as the second, etc. I literally constantly try to remember which of `sub()`, `gsub()`, `grep()`, etc. do what, and always have to refer to the help docs for that + where I place the arguments in the calls!

Reply:
Yes that looks so much better than regex! I also struggle big time. I'll leave for now since I'll remove this anyways once I rename all workflows to match the indicator IDs.